lib/cc/analyzer/formatters/plain_text_formatter.rb

```diff
@@ -42,11 +42,11 @@ def engine_running(engine, &block)
           end
         end
 
-        def failed(output)
-          spinner.stop("Failed")
-          puts colorize("\nAnalysis failed with the following output:", :red)
+        def errored(output)
+          spinner.stop("Errored")
+          puts colorize("\nAnalysis errored with the following output:", :red)
           puts output
-          exit 1
+          exit 2
```

Comment:
We could, probably: I think keeping them unique at the CLI level is desirable, though, if we're going to differentiate them at all. The grouping-by-10 suggestion is primarily so that each category would still be contiguous if we decided to add a new code to a category.